src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,29 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
```

Comment:
MachineProducer is parameterized class.
Raw type here.
```suggestion
public class BulldozerProducer implements MachineProducer<> {
```
fix everywhere